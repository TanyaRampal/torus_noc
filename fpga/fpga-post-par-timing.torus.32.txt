Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
-----------------------------------------------------------------------------------------
| Tool Version : Vivado v.2022.1 (lin64) Build 3526262 Mon Apr 18 15:47:01 MDT 2022
| Date         : Fri Dec  6 14:33:54 2024
| Host         : ECEUBUNTU2 running 64-bit Red Hat Enterprise Linux release 8.10 (Ootpa)
| Command      : report_timing -file fpga-post-par-timing.torus.32.txt
| Design       : torus
| Device       : 7z020-clg400
| Speed File   : -1  PRODUCTION 1.12 2019-11-22
-----------------------------------------------------------------------------------------

Timing Report

Slack (MET) :             1.976ns  (required time - arrival time)
  Source:                 ys[1].xs[3].torus_switch_xy/w_in_x_reg_reg[1]/C
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@3.000ns period=6.000ns})
  Destination:            ys[1].xs[3].client_xy/i_x_r_reg[1]/D
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@3.000ns period=6.000ns})
  Path Group:             clk
  Path Type:              Setup (Max at Slow Process Corner)
  Requirement:            6.000ns  (clk rise@6.000ns - clk rise@0.000ns)
  Data Path Delay:        3.688ns  (logic 0.792ns (21.475%)  route 2.896ns (78.525%))
  Logic Levels:           2  (LUT4=1 LUT6=1)
  Clock Path Skew:        -0.049ns (DCD - SCD + CPR)
    Destination Clock Delay (DCD):    0.924ns = ( 6.924 - 6.000 ) 
    Source Clock Delay      (SCD):    0.973ns
    Clock Pessimism Removal (CPR):    0.000ns
  Clock Uncertainty:      0.035ns  ((TSJ^2 + TIJ^2)^1/2 + DJ) / 2 + PE
    Total System Jitter     (TSJ):    0.071ns
    Total Input Jitter      (TIJ):    0.000ns
    Discrete Jitter          (DJ):    0.000ns
    Phase Error              (PE):    0.000ns

    Location             Delay type                Incr(ns)  Path(ns)    PBlock               Netlist Resource(s)
  -------------------------------------------------------------------    ----------------------------------------
                         (clock clk rise edge)        0.000     0.000 r                       
                                                      0.000     0.000 r                       clk (IN)
                         net (fo=527, unset)          0.973     0.973                         ys[1].xs[3].torus_switch_xy/clk
    SLICE_X82Y93         FDRE                                         r  poly3_1_sw           ys[1].xs[3].torus_switch_xy/w_in_x_reg_reg[1]/C
  -------------------------------------------------------------------    ----------------------------------------
    SLICE_X82Y93         FDRE (Prop_fdre_C_Q)         0.518     1.491 r  poly3_1_sw           ys[1].xs[3].torus_switch_xy/w_in_x_reg_reg[1]/Q
                         net (fo=11, routed)          1.809     3.300                         ys[1].xs[3].torus_switch_xy/w_msg[35]
    SLICE_X62Y95         LUT6 (Prop_lut6_I3_O)        0.124     3.424 r  poly3_1_sw           ys[1].xs[3].torus_switch_xy/waiting_for_ack_i_5__6/O
                         net (fo=7, routed)           0.704     4.128                         ys[1].xs[3].torus_switch_xy/i_x_r_reg[1]
    SLICE_X62Y95         LUT4 (Prop_lut4_I2_O)        0.150     4.278 r  poly3_1_sw           ys[1].xs[3].torus_switch_xy/i_x_r[1]_i_1__6/O
                         net (fo=1, routed)           0.383     4.661                         ys[1].xs[3].client_xy/i_x_r_reg[1]_1[1]
    SLICE_X62Y95         FDRE                                         r  poly3_1_cli          ys[1].xs[3].client_xy/i_x_r_reg[1]/D
  -------------------------------------------------------------------    ----------------------------------------

                         (clock clk rise edge)        6.000     6.000 r                       
                                                      0.000     6.000 r                       clk (IN)
                         net (fo=527, unset)          0.924     6.924                         ys[1].xs[3].client_xy/clk
    SLICE_X62Y95         FDRE                                         r  poly3_1_cli          ys[1].xs[3].client_xy/i_x_r_reg[1]/C
                         clock pessimism              0.000     6.924                           
                         clock uncertainty           -0.035     6.889                           
    SLICE_X62Y95         FDRE (Setup_fdre_C_D)       -0.252     6.637    poly3_1_cli            ys[1].xs[3].client_xy/i_x_r_reg[1]
  -------------------------------------------------------------------
                         required time                          6.637                           
                         arrival time                          -4.661                           
  -------------------------------------------------------------------
                         slack                                  1.976                           




