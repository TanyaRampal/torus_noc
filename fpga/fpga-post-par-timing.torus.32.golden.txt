Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
-----------------------------------------------------------------------------------------
| Tool Version : Vivado v.2022.1 (lin64) Build 3526262 Mon Apr 18 15:47:01 MDT 2022
| Date         : Tue Nov 26 13:45:32 2024
| Host         : ECEUBUNTU2 running 64-bit Red Hat Enterprise Linux release 8.10 (Ootpa)
| Command      : report_timing -file fpga-post-par-timing.torus.32.txt
| Design       : torus
| Device       : 7z020-clg400
| Speed File   : -1  PRODUCTION 1.12 2019-11-22
-----------------------------------------------------------------------------------------

Timing Report

Slack (MET) :             3.290ns  (required time - arrival time)
  Source:                 ys[3].xs[2].cli/packet_num_reg[7]/C
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@4.500ns period=9.000ns})
  Destination:            ys[3].xs[2].cli/i_x_r_reg[0]/CE
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@4.500ns period=9.000ns})
  Path Group:             clk
  Path Type:              Setup (Max at Slow Process Corner)
  Requirement:            9.000ns  (clk rise@9.000ns - clk rise@0.000ns)
  Data Path Delay:        5.421ns  (logic 1.984ns (36.596%)  route 3.437ns (63.404%))
  Logic Levels:           6  (CARRY4=4 LUT2=1 LUT3=1)
  Clock Path Skew:        -0.049ns (DCD - SCD + CPR)
    Destination Clock Delay (DCD):    0.924ns = ( 9.924 - 9.000 ) 
    Source Clock Delay      (SCD):    0.973ns
    Clock Pessimism Removal (CPR):    0.000ns
  Clock Uncertainty:      0.035ns  ((TSJ^2 + TIJ^2)^1/2 + DJ) / 2 + PE
    Total System Jitter     (TSJ):    0.071ns
    Total Input Jitter      (TIJ):    0.000ns
    Discrete Jitter          (DJ):    0.000ns
    Phase Error              (PE):    0.000ns

    Location             Delay type                Incr(ns)  Path(ns)    PBlock               Netlist Resource(s)
  -------------------------------------------------------------------    ----------------------------------------
                         (clock clk rise edge)        0.000     0.000 r                       
                                                      0.000     0.000 r                       clk (IN)
                         net (fo=1071, unset)         0.973     0.973                         ys[3].xs[2].cli/clk
    SLICE_X106Y67        FDRE                                         r  poly2_3_cli          ys[3].xs[2].cli/packet_num_reg[7]/C
  -------------------------------------------------------------------    ----------------------------------------
    SLICE_X106Y67        FDRE (Prop_fdre_C_Q)         0.456     1.429 r  poly2_3_cli          ys[3].xs[2].cli/packet_num_reg[7]/Q
                         net (fo=4, routed)           0.821     2.250                         ys[3].xs[2].cli/regulator/out[1]
    SLICE_X107Y68        LUT2 (Prop_lut2_I0_O)        0.124     2.374 r  poly2_3_cli          ys[3].xs[2].cli/regulator/waiting_for_ack_i_22/O
                         net (fo=1, routed)           0.000     2.374                         ys[3].xs[2].cli/regulator/waiting_for_ack_i_22_n_0
    SLICE_X107Y68        CARRY4 (Prop_carry4_S[0]_CO[3])
                                                      0.532     2.906 r  poly2_3_cli          ys[3].xs[2].cli/regulator/waiting_for_ack_reg_i_13__1/CO[3]
                         net (fo=1, routed)           0.000     2.906                         ys[3].xs[2].cli/regulator/waiting_for_ack_reg_i_13__1_n_0
    SLICE_X107Y69        CARRY4 (Prop_carry4_CI_CO[3])
                                                      0.114     3.020 r  poly2_3_cli          ys[3].xs[2].cli/regulator/waiting_for_ack_reg_i_8__1/CO[3]
                         net (fo=1, routed)           0.000     3.020                         ys[3].xs[2].cli/regulator/waiting_for_ack_reg_i_8__1_n_0
    SLICE_X107Y70        CARRY4 (Prop_carry4_CI_CO[3])
                                                      0.114     3.134 r  poly2_3_cli          ys[3].xs[2].cli/regulator/waiting_for_ack_reg_i_5__13/CO[3]
                         net (fo=1, routed)           0.000     3.134                         ys[3].xs[2].cli/regulator/waiting_for_ack_reg_i_5__13_n_0
    SLICE_X107Y71        CARRY4 (Prop_carry4_CI_CO[0])
                                                      0.271     3.405 r  poly2_3_cli          ys[3].xs[2].cli/regulator/waiting_for_ack_reg_i_3__13/CO[0]
                         net (fo=6, routed)           1.288     4.693                         ys[3].xs[2].cli/_12_
    SLICE_X106Y65        LUT3 (Prop_lut3_I0_O)        0.373     5.066 r  poly2_3_cli          ys[3].xs[2].cli/waiting_for_ack_i_1__13/O
                         net (fo=5, routed)           1.329     6.394                         ys[3].xs[2].cli/_10_
    SLICE_X101Y57        FDRE                                         r  poly2_3_cli          ys[3].xs[2].cli/i_x_r_reg[0]/CE
  -------------------------------------------------------------------    ----------------------------------------

                         (clock clk rise edge)        9.000     9.000 r                       
                                                      0.000     9.000 r                       clk (IN)
                         net (fo=1071, unset)         0.924     9.924                         ys[3].xs[2].cli/clk
    SLICE_X101Y57        FDRE                                         r  poly2_3_cli          ys[3].xs[2].cli/i_x_r_reg[0]/C
                         clock pessimism              0.000     9.924                           
                         clock uncertainty           -0.035     9.889                           
    SLICE_X101Y57        FDRE (Setup_fdre_C_CE)      -0.205     9.684    poly2_3_cli            ys[3].xs[2].cli/i_x_r_reg[0]
  -------------------------------------------------------------------
                         required time                          9.684                           
                         arrival time                          -6.394                           
  -------------------------------------------------------------------
                         slack                                  3.290                           




